src/FormatNumber/Parser.elm

```diff
@@ -378,3 +442,54 @@ parse locale original =
                 | prefix = locale.zeroPrefix
                 , suffix = locale.zeroSuffix
             }
+
+
+{-| Given a `Locale` parses a `String` into a `Maybe Float`
+-}
+parseString : Locale -> String -> Maybe Float
+parseString locale value =
+    let
+        isNegative : Bool
+        isNegative =
+            String.startsWith locale.negativePrefix value
+                && String.endsWith
+                    locale.negativeSuffix
+                    value
```

Comment:
Is this formatting what Elm Format suggests? Weird, but if it is Elm Format, we accept it hahaha…

Reply:
It is from Elm format.